### PR TITLE
Implement new design

### PR DESCRIPTION
**Pull request by caseypt**
This is a rough implementation of the new design. There are several follow-up issues: https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/686, https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/687, https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/688, https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/689, https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/690, https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/691, https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/692.

**Testing**
- In a bash window, start a web server in the prototype folder, and visit the site in the browser.
- Run the framework and open it up in a browser. 
- Compare the two sites and verify the general structure is same and that the styles are roughly implemented.

image

Connects to #684

### CODE REVIEW

File: src/GeositeFramework/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -286,10 +288,10 @@
     </script>
 
     <script type="text/template" id="template-sidebar-plugin">
-            <a class="plugin-launcher i18n" data-i18n="[title]<%=fullName%>" href="javascript:;">
-                <img src="<%- pluginSrcFolder %>/icon<%= selected ? '_active' : ''%>.png">
-                <div class="i18n" data-i18n="<%- pluginObject.toolbarName %>"><%- pluginObject.toolbarName %></div>
-            </a>
+        <button class="nav-apps-button" i18n" data-i18n="[title]<%=fullName%>">
```
Comment (mmcfarland) [inline src/GeositeFramework/Views/Home/Index.cshtml]: I suspect this is the same in the current branch, but when these plugin items are rendered they appear to get wrapped in an `<li>` tag which isn't present in the wireframes.  It looks like they have some attributes on them that is important, but they don't seem to get written inside of a `<ul>` tag.  If it's not too costly, consider adding the appropriate container element.

screenshot 2016-10-17 at 1 26 17 pm

File: src/GeositeFramework/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -525,23 +527,42 @@
...
-            <section>
-                <ul class="right">
-                    @RenderLinks(Model.HeaderLinks)
-                </ul>
-            </section>
-        </nav>
-    </div>
+        </div>
+        <div class="dropdown header-dropdown nav-locations">
+            <button class="button dropdown-toggle nav-main-button" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
```
Comment (mmcfarland) [inline src/GeositeFramework/Views/Home/Index.cshtml]: `dropdownMenu1` from the prototype is used as an id on a few different elements in this PR.  Consider changing the id/labeledby sets to more meaningful and unique names.

File: src/GeositeFramework/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -82,8 +83,9 @@
     <link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
     <link rel="stylesheet" href="css/pageguide.min.css"/>
     <link rel="stylesheet" href="css/app.css">
-    <link rel="stylesheet" href="css/jquery.dropdown.css"/>
+    <link rel="stylesheet" href="css/main.css">
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:500,900" rel="stylesheet">
```
Comment (mmcfarland) [inline src/GeositeFramework/Views/Home/Index.cshtml]: Doesn't look like this font is getting picked up by any of the elements.  I think we have a more specific style set in the old css.

File: src/GeositeFramework/Views/Home/Index.cshtml (hunk 1 above)
Comment (caseypt) [inline src/GeositeFramework/Views/Home/Index.cshtml]: Good catch. I just ran across this while working on #686. I'm going to fix it there as it has implications in the plugin code.

File: src/GeositeFramework/Views/Home/Index.cshtml (hunk 3 above)
Comment (caseypt) [inline src/GeositeFramework/Views/Home/Index.cshtml]: Thanks. Fixed in 9f62d89.

image

File: src/GeositeFramework/Views/Home/Index.cshtml (hunk 2 above)
Comment (caseypt) [inline src/GeositeFramework/Views/Home/Index.cshtml]: Addressed in 	009eeee.

**Comment by caseypt** [comment]
Thanks for the review.

**Pull request merged**